core/dbt/adapters/base/impl.py

```diff
@@ -1518,12 +1608,14 @@ def render_model_constraint(cls, constraint: ModelLevelConstraint) -> Optional[s
         else:
             return None
 
```

Comment:
should this also be a classmethod? similar to its override in plugins/postgres/dbt/adapters/postgres/impl.py

Reply:
My ambivalence around using static-methods/classmethods is that they make testing difficult. I don't mind using them for pure functions, but when working on a MockAdapter class for testing (not part of this review) I immediately hit the problem that I have to declare an entirely new class to override the features reported by the mock.

That said, enough people have voted for classmethod here that I'll change it for both

Reply:
Done.